packages/ldap-auth/src/components/LoginPage/Form.tsx

```diff
@@ -35,85 +35,82 @@ passwordSchema.is().min(4).not().spaces();
+    }
 
-  useEffect(() => {
-    const keyDownHandler = (event: {
-      key: string;
-      preventDefault: () => void;
-    }) => {
-      if (event.key === 'Enter') {
-        event.preventDefault();
-        onClick();
-      }
-    };
+    useEffect(() => {
+        const keyDownHandler = (event: {
+            key: string;
+            preventDefault: () => void;
+        }) => {
+            if (event.key === 'Enter') {
+                event.preventDefault();
+                onClick();
+            }
+        };
 
-    document.addEventListener('keydown', keyDownHandler);
+        document.addEventListener('keydown', keyDownHandler);
 
-    return () => {
-      document.removeEventListener('keydown', keyDownHandler);
-    };
-  });
+        return () => {
+            document.removeEventListener('keydown', keyDownHandler);
+        };
+    });
 
-  return (
-    <Page themeId="tool">
-      <Content>
-        <Container maxWidth="sm">
-          <Typography variant="h5" component="h2" className={classes.paperHead}>
-            LDAP login page
-          </Typography>
-          <Paper elevation={4} className={classes.paper}>
-            <TextField
-              required
-              label="LDAP Name"
-              onChange={e => setUsername(e.target.value)}
-              value={username}
-              id="username"
-              error={uError}
-              helperText={`Usually it's  the first letter of your name followed by the surname, e.g. Dario Rossi 👉 drossi`}
-              fullWidth
-              size="small"
-              margin="dense"
-            />
-            <TextField
-              required
-              label="Password"
-              onChange={e => setPassword(e.target.value)}
-              value={password}
-              id="password"
-              type="password"
-              error={pError}
-              fullWidth
-              size="small"
-              margin="dense"
-            />
-            <Button
-              variant="contained"
-              color="primary"
-              size="large"
-              fullWidth
-              onClick={onClick}
-              type="submit"
-              style={{ marginTop: 16 }}
-            >
-              Login
-            </Button>
-          </Paper>
-        </Container>
-      </Content>
-    </Page>
-  );
+    return (
+        <Page themeId="tool">
+            <Content>
+                <Container maxWidth="sm">
+                    <Paper elevation={4} className={classes.paper}>
+                        <TextField
+                            required
+                            label="LDAP Name"
+                            onChange={(e) => setUsername(e.target.value)}
+                            value={username}
+                            id="username"
+                            error={uError}
+                            helperText={`Usually it's  the first letter of your name followed by the surname, e.g. Dario Rossi 👉 drossi`}
```

Comment:
This can be a variable passed as input into the component props.

Reply:
This can also be blank, I don't think it's needed to add an explanation on usernames, everyone should knows his own

Reply:
Yes, but because this is an internal tool, it can be helpful to explain some information... like, where you can take it... From my point of view, adding an optional variable can be beneficial.

Reply:
We can easily do this by patching it, or, we can add this to our backlog when we do a way to customize the look of the form, if you have a fast way to do it cleanly be my guest and do a little PR :)